src_analysis/change_utils.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
```

Comment:
The purpose of this module was to be aware of things like gcb/* and docker/* so that fuzzer_dependencies and benchmark_dependencies could ignore them. It is unnecessary if we go with the always_build_fuzzer approach. Should I remove and use those modules directly?